projects/tangible/index.js

```diff
@@ -0,0 +1,15 @@
+const sdk = require('@defillama/sdk');
+const utils = require('../helper/utils');
+const { transformPolygonAddress } = require('../helper/portedTokens');
+const USDR_TOKEN_CONTRACT = '0xb5DFABd7fF7F83BAB83995E72A52B97ABb7bcf63';
+
+async function tvlCalc() {
+
+  const result = await utils.fetchURL("https://api.tangible.store/tvl");
```

Comment:
hi @veljkoTNFT , thanks for the PR

can the tvl be computed on chain instead? this is a bit opaque,  I have no idea what is being included in your tvl

Maybe you can fetch the minted TNFTs and their price from oracle and compute tvl?

Reply:
Hi @g1nt0ki. I guess I can. I wanted to avoid learning yet another sdk for interacting with blockchain :D

Reply:
@g1nt0ki  can you help me understand how do I sum numbers that I get? because they are treated as strings and simple + will just concat. How can I get BigNumber math?